benchmark/utils.go

```diff
@@ -55,3 +56,55 @@ func GetImageListFromCsv(csvLoc string) ([]ImageDescriptor, error) {
 	}
 	return images, nil
 }
+
+func GetCommitHash() (string, error) {
+	cmd := exec.Command("git", "rev-parse", "HEAD")
```

Comment:
nit: This can be a follow up, but this can be wrong if you don't build right before running the benchmarks. I wonder if we should embed a version into the benchmark binary instead like we do for the SOCI CLI/snapshotter.

Reply:
Understood, Would like to discuss this more and possibly change it in the next code commit.